src/main/java/com/study/platform/blog/service/FeedApiServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package com.study.platform.blog.service;
+
+import com.study.platform.blog.domain.Feed;
+import com.study.platform.blog.domain.FeedRepository;
+import com.study.platform.blog.service.dto.FeedDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Component;
+
+@RequiredArgsConstructor
+@Component
```

Comment:
```suggestion
@Service
```
Service는 Component가 아닌 Service로 명시해 빈을 등록하는게 더 좋다고 생각합니다 :)